README.md

```diff
@@ -1,13 +1,20 @@
 # Mongo API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Model a database using Mongoose and produce restful Api.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I used same Netflix dataset as in previous project. Having difficulties finding motivation I'm glad I didn't just give up and minmimum requirements are met.
+If I had more time I would add a title with reg ex for enabling intuitive search, as well as a year endpoint and limit the amount of movies per page. Also change the director ref type as it would make more sense to have an object id type per country instead of director.
+Add a title with reg ex for enabling intuitive search, as well as a year endpoint.
+
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://simple-mongo-db.herokuapp.com/
+
+Endpoints:
+ https://simple-mongo-db.herokuapp.com/shows   /returns whole array of netflixdata 
+ https://simple-mongo-db.herokuapp.com/shows/81171862    /returns single show based on json.show-id 
+
+ https://simple-mongo-db.herokuapp.com/shows/director/546f6b61204d634261726f72   /returns show(s) by director db created objectId 
```

Comment:
I liked the fact that you are using an endpoint with own generated db object id!